### PR TITLE
Create celery task to reset every events' is_ready and bumped status

### PR DESCRIPTION
**Pull request by norbye**
Set to run every day at 04:00

Intent to resolve events that have been invalidated during celery down-time

### CODE REVIEW

File: lego/apps/events/tasks.py [hunk 1]
```diff
@@ -341,6 +341,20 @@ def save_and_notify_payment(self, result, registration_id, logger_context=None):
     )
 
 
+@celery_app.task(serializer="json", bind=True, base=AbakusTask)
+def set_all_events_ready_and_bump(self, logger_context=None):
+    """Task to bump all tasks to is_read=True in case of error"""
```
Comment (erlingfn) [inline lego/apps/events/tasks.py:346]: Should this not say `is_ready=True`?

File: lego/apps/events/tasks.py [hunk 2]
```diff
@@ -341,6 +341,20 @@ def save_and_notify_payment(self, result, registration_id, logger_context=None):
...
+    self.setup_logger(logger_context)
+
+    # Find all events that are set to "is_ready=False" and are not awaiting automatic celery task (have just been edited)
+    now = timezone.now()
+    events = Event.objects.filter(is_ready=False).filter(
```
Comment (erlingfn) [inline lego/apps/events/tasks.py:351]: Maybe name this `bad_events` or something?

**Review by erlingfn — APPROVED** [review]
lgtm

**Review by SmithPeder — APPROVED** [review]
LGTM 🌮

**Pull request merged**